e2e-tests/playwright/package.json

```diff
@@ -1,33 +1,34 @@
 {
     "scripts": {
-        "test": "PW_SNAPSHOT_ENABLE=true playwright test",
+        "test": "NODE_OPTIONS='--no-experimental-fetch' PW_SNAPSHOT_ENABLE=true playwright test",
```

Comment:
can we use `cross-env` here for cross platform environmental support?

Reply:
Sure, now added. Thanks Zubair!